src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -754,7 +775,7 @@ public ResponseEntity<Void> stopSecurityAnalysis(@Parameter(description = "Study
     public ResponseEntity<List<ReporterModel>> getNodeReport(@Parameter(description = "Study uuid") @PathVariable("studyUuid") UUID studyUuid,
                                                              @Parameter(description = "Node uuid") @PathVariable("nodeUuid") UUID nodeUuid,
                                                              @Parameter(description = "Node only report") @RequestParam(value = "nodeOnlyReport", required = false, defaultValue = "true") boolean nodeOnlyReport) {
-        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getNodeReport(studyUuid, nodeUuid, nodeOnlyReport));
```

Comment:
remove studyUuid from PathVariable if we don't using it

Reply:
Fixed